packages/core/parcel-bundler/src/assets/JSAsset.js

```diff
@@ -82,9 +85,14 @@ class JSAsset extends Asset {
     walk.ancestor(this.ast, collectDependencies, this);
   }
 
+  matchSourceMappingURL() {
```

Comment:
Why did this get moved into a seperate function?

Reply:
only for testing purposes, I didn't want to test against `this.contents` directly, but rather against an (internal) API that `loadSourceMap` uses. I thought this way, if someone decides to adjust `matchSourceMappingURL`, the test would still kick in.